docs/get-started-dapr-actor.md

```diff
@@ -58,7 +58,7 @@ Update the csproj file to target .NET Core 3.0
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Dapr.Actors" Version="0.1.0-preview01" />
+    <PackageReference Include="Dapr.Actors" Version="0.7.0-preview01" />
```

Comment:
```suggestion
    <PackageReference Include="Dapr.Actors" Version="0.8.0-preview01" />
```

Reply:
I think it may be better to say the latest as a comment, then "for example 0.8.0-preview-01" otherwise these docs will need to be updated with every release.

Reply:
...at least until the first stable release 🙂 It's definitely a roadblock for newcomers. I had a hard time figuring this out.

Reply:
Do you think we could run a PowerShell command for the version number?

Reply:
You can add the 0.8.0-preview01 but add a note that use latest version for the package from nuget.org

Reply:
This Powershell command works with preview versions
```powershell
dotnet add package Dapr.Actors -v "$((Find-Package -Name Dapr.Actors -AllVersions -AllowPrereleaseVersions | Select -First 1).Version)"
```

Reply:
@LayZeeDK Thanks for trying the powershell. It would not work on mac and linux unless users have powershell core isntalled. Its ok to add the 0.8.0-preview01 but add a note that use latest version for the package from nuget.org

Reply:
I've updated the version to 0.9.0-preview01 and above each occurance added a note to use the latest version from nuget.